src/js/components/Institutions.jsx

```diff
@@ -109,26 +155,47 @@ export default class Institution extends Component {
         pathname={this.props.location.pathname}
         userName={this.props.user.profile.name} />
       <div id="main-content" className="usa-grid">
-        <UserHeading period="2017" userName={this.props.user.profile.name} />
+        <UserHeading
+          period="2017"
+          userName={this.props.user.profile.name} />
         <div className="usa-width-two-thirds">
           {this.props.filings.map((filingObj, i) => {
             const filing = filingObj.filing
+            const latestSubmissionStatus = filingObj.submissions[0].status || null
```

Comment:
This makes me slightly uncomfortable since I can't find the code in the backend that gives a hard guarantee that the latest submission (highest sequence number) will always be at index 0. I'd like not to burn extra cycles here, but correctness is paramount.

@schbetsy could you weigh on on sorting of submissions?